.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
```

Comment:
Can you also recommend a vscode extension that runs prettier? (unless it has that built-in)

Reply:
Why do we track personal editor choices globally? Is this usual practice? 

(It's hard to read tone over text, so I'll be explicit: I'm aware this issue isn't worth the attention, I'm just curious as to whether this is a common practice. I was under the impression teams usually have a strict moratorium on personal configuration files.)

Reply:
It varies. Open source projects usually don't, companies usually do. In this case it's just a suggestion rather than actual settings, and most of us will be using vscode anyway, so I'm fine with it.